client/ayon_nuke/hooks/log_no_colors.py

```diff
@@ -0,0 +1,10 @@
+from ayon_applications import PreLaunchHook
+
+
+class LogNoColorsHook(PreLaunchHook):
+    """
+    Disable colored logs in Nuke because the console doesn't interpret the control sequences
+    """
+
+    def execute(self):
+        self.launch_context.env["AYON_LOG_NO_COLORS"] = "1"
```

Comment:
To match other hosts I think it's better to set this in the `Addon.add_implementation_envs` method.

For example, see: https://github.com/ynput/ayon-aftereffects/blob/117d988bf8793eb289edf242b1b5efd41b6130cc/client/ayon_aftereffects/addon.py#L15-L23

Reply:
All right did that!